### PR TITLE
Updated the section "Installation from source"

### PR DESCRIPTION
**Pull request by Mauladen**
Updated the section "Installation from Sourse" for a more understandable and clean build

- Added Go installation
- Build from under the user "git"

### CODE REVIEW

File: docs/content/doc/installation/from-source.en-us.md [hunk 1]
```diff
@@ -15,18 +15,63 @@ menu:
 
 # Installation from source
 
-This section will not include basic [installation instructions](https://golang.org/doc/install).
+## Dependencies
 
-**Note**: Go version 1.8 or higher is required
+**Note**: [Go Programming Language](http://golang.org): Version >= 1.8
```
Comment (techknowlogick) [inline docs/content/doc/installation/from-source.en-us.md]: The minimum supported version of Go will soon be 1.9

File: docs/content/doc/installation/from-source.en-us.md [hunk 2]
```diff
@@ -15,18 +15,63 @@ menu:
...
+   --disabled-password \
+   --home /home/git \
+   git
+```
+
+## Installing Go
+
+If your system's Go matches the requirements, please skip this section.
+
+### Download
```
Comment (techknowlogick) [inline docs/content/doc/installation/from-source.en-us.md:39]: I don't think we need a tutorial on installing Go, as https://golang.org/doc/install will always be up to date, and we can just link to that.

File: docs/content/doc/installation/from-source.en-us.md [hunk 3]
```diff
@@ -35,15 +80,15 @@ with master, skip to the [build section](#build).
 
 To work with tagged releases, the following commands can be used:
 ```
-git branch -a
-git checkout v1.0
+$ git branch -a
+$ git checkout v1.0
```
Comment (techknowlogick) [inline docs/content/doc/installation/from-source.en-us.md]: Maybe a newer tag (perhaps 1.4.2) should be used?

File: docs/content/doc/installation/from-source.en-us.md (hunk 2 above)
Comment (Mauladen) [reply docs/content/doc/installation/from-source.en-us.md:39]: But to run Gitea created a new user, and then I added the construction using the user git

**Pull request closed without merging**